docs/learnpaths/kubewarden-user.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_label: User
```

Comment:
Maybe I would rename it to "Policy user", or "Policy consumer" everywhere, User maybe is too broad. WDYT?

Reply:
Yes, agreed. I think Policy user will be clear enough.